lib/ansible/modules/remote_management/hpilo/hpilo_boot.py

```diff
@@ -75,6 +75,12 @@
     - As a safeguard, without force, hpilo_boot will refuse to reboot a server that is already running.
     default: no
     choices: [ "yes", "no" ]
+  ssl_version:
+    description:
+      - Change the ssl_version used
```

Comment:
Just to nitpick, I would have indented this one as the others, and have a trailing dot.
But those are not showstoppers.